sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -48,6 +64,77 @@ export function isTracingDisabled(): boolean {
   return Boolean(azureTracingDisabledValue);
 }
 
+/**
+ * Maintains backwards compatibility with the previous `OperationTracingOptions` which included `SpanOptions`
```

Comment:
Should we add more information related to which versions this might be relevant to?